test/specs/scale.time.tests.js

```diff
@@ -297,8 +297,8 @@ describe('Time scale tests', function() {
 		var xScale = chart.scales.xScale0;
 
 		// Counts down because the lines are drawn top to bottom
-		expect(xScale.ticks[0].value).toBe('Jan 2');
-		expect(xScale.ticks[1].value).toBe('May 8');
+		expect(xScale.ticks[0].value).toEqualOneOf(['Jan 1', 'Jan 2']);
```

Comment:
I have been able to reproduce this issue locally by modifying my local time zone. Changing the parser (as above) fixes the issue for me.